src/com/superiornetworks/icarus/commands/Command_dellogin.java

```diff
@@ -0,0 +1,41 @@
+package com.superiornetworks.icarus.commands;
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.ICM_SqlHandler;
+import com.superiornetworks.icarus.ICM_Utils;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+
+@CommandParameters(name="dellogin",description="Remove your custom login message.",usage="/dellogin <message>",rank=ICM_Rank.Rank.SUPER)
+public class Command_dellogin
+{
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args)
+    {
+        if(args.length == 0)
+        {
+            return false;
+        }
+        String combined = StringUtils.join(ArrayUtils.subarray(args, 0, args.length), " ");
+        try
+        {
+            PreparedStatement statement = ICM_SqlHandler.getConnection().prepareStatement("DELETE FROM `players` WHERE `loginMessage` = ? WHERE `playerName` = ?");
```

Comment:
I believe that this deletes the whole player-data of the player. I think this because you are deleting from the table itself. Instead of doing this, just set their login message to ''.